Connection.php

```diff
@@ -555,3 +555,9 @@ public function close($code = self::CLOSE_NORMAL, $reason = null)
         return $connection->disconnect();
     }
 }
+
+/**
+ * Register socket wrappers
+ */
+HoaSocket\Transport::registerWrapper('ws', ['Hoa\Websocket\Socket', 'createFromUri']);
```

Comment:
Please, move this in `Socket.php`, see https://github.com/hoaproject/Websocket/pull/47/files#r46777215 for more explanations.